website/docs/d/pi_volume.html.markdown

```diff
@@ -48,19 +48,23 @@ Review the argument references that you can specify for your data source.
 
 In addition to all argument reference list, you can access the following attribute references after your data source is created.
 
-- `auxiliary` - (Boolean) Indicates if the volume is auxiliary or not.
+- `auxiliary` - (Boolean) Indicates if the volume is auxiliary.
 - `auxiliary_volume_name` - (String) The auxiliary volume name.
 - `bootable` -  (Boolean) Indicates if the volume is boot capable.
 - `consistency_group_name` - (String) Consistency group name if volume is a part of volume group.
+- `creation_date` - (String) Date of volume creation.
 - `disk_type` - (String) The disk type that is used for the volume.
+- `freeze_time` - (String) Time volume last replicated.
 - `group_id` - (String) The volume group id in which the volume belongs.
 - `id` - (String) The unique identifier of the volume.
 - `io_throttle_rate` - (String) Amount of iops assigned to the volume.
+- `last_update_date` - (String) The date when the volume last updated.
 - `master_volume_name` - (String) The master volume name.
 - `mirroring_state` - (String) Mirroring state for replication enabled volume.
 - `primary_role` - (String) Indicates whether `master`/`auxiliary` volume is playing the primary role.
 - `replication_enabled` - (Boolean) Indicates if the volume should be replication enabled or not.
 - `replication_status` - (String) The replication status of the volume.
+- `replication_sites` - (List) List of replication sites for disaster recovery for volume. 
```

Comment:
replication_sites and replication_status is not sorted correctly.

Reply:
Sorted in latest commit.